patch/0001-armhf-default-config-for-the-sonic-patches.patch

```diff
@@ -0,0 +1,82 @@
+From 3a09f684413faf1c4f7d21d7d79914b54e087cf6 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 10:48:29 +0530
+Subject: [PATCH] armhf default config for the sonic patches
+
+---
+ debian/build/build_armhf_none_armmp/.config | 10 ++++++++++
+ 1 file changed, 10 insertions(+)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index 8f02d03..32a2698 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -1501,6 +1501,7 @@ CONFIG_NET_ACT_POLICE=m
+ CONFIG_NET_ACT_GACT=m
+ CONFIG_GACT_PROB=y
+ CONFIG_NET_ACT_MIRRED=m
++# CONFIG_NET_ACT_SAMPLE is not set
```

Comment:
why do you need this? you enabled it in the kernel-enable-psample-and_act_ample_drivers.patch.

Reply:
Explained the reason in https://github.com/Azure/sonic-linux-kernel/pull/102#discussion_r339895779